project.conf

```
@@ -35,6 +35,13 @@ options:
     - i686
     - x86_64
 
+# Artifact servers.
+#
+# These are used to fetch built artifacts when available.
+#
+artifacts:
+  - url: https://bstcache.endlessos.org
```

Tiny nitpick - the other lists in here are formatted without the leading spaces.

```suggestion
- url: https://bstcache.endlessos.org
```

---

Thanks. I ended up squashing this.